Directory.Packages.props

```diff
@@ -143,7 +143,7 @@
     <PackageVersion Include="Microsoft.AspNetCore.Authorization" Version="9.0.2" />
 
     <!-- dotnet/extensions repository -->
-    <PackageVersion Include="Microsoft.Extensions.Http.Resilience" Version="9.1.0" />
+    <PackageVersion Include="Microsoft.Extensions.Http.Resilience" Version="9.2.0" />
```

Comment:
Note: bumping that dependency was required as OpenIddict references the 9.2.0 version of that package.